src/common/actions/repositories.js

```diff
@@ -0,0 +1,68 @@
+import 'isomorphic-fetch';
+
+import conf from '../helpers/config';
+
+const BASE_URL = conf.get('BASE_URL');
+
+export const FETCH_REPOSITORIES = 'FETCH_REPOSITORIES';
+export const FETCH_REPOSITORIES_ERROR = 'FETCH_REPOSITORIES_ERROR';
```

Comment:
We use the same pattern in other actions/reducers, so full refactoring would be needed.
We have FSA card in backlog, so let's do it separately.